### PR TITLE
add id use for verify command

### PR DESCRIPTION
**Pull request by Naxey**
:D

### CODE REVIEW

**Comment by maxwai** [comment]
Please some explanation why this change is needed. What use case this solves.

File: cogs/moderator.py [hunk 1]
```diff
@@ -76,7 +77,12 @@ async def verify(self, ctx: Context, member):  # parameter only for pretty help.
...
         global verified
         try:
-            member: Union[Member, User] = ctx.message.mentions[0]
+            if ctx.message.mentions:
+                member: Union[Member, User] = ctx.message.mentions[0]
+            elif ctx.message.content.split(" ")[1].isnumeric():
+                member: Union[Member, User] = self.bot.get_user(
+                    int(ctx.message.content.split(" ")[1])
+                )
         except IndexError:
```
Comment (maxwai) [inline cogs/moderator.py:86]: This can be easily gotten by using the `member` attribute of the function. Since no Type is given, discord.py will accept any Type.

Also the `ValueError` exception should be in the catch statement since we don't want any runaway exceptions.

Also you should use the `Guild` Attribute of the `Context` Attribute to get a Member, this is more efficient as not every Guild needs to be checked.

```suggestion
                member: Union[Member, User] = ctx.guild.get_member(int(member))
        except (IndexError, ValueError):
```

File: cogs/moderator.py [hunk 2]
```diff
@@ -65,7 +65,8 @@ def cog_unload(self):
     @command(help="Verify a mentioned user")
     @bot_chat(bot_channels)
     @has_role_plus(moderator)
-    async def verify(self, ctx: Context, member):  # parameter only for pretty help.
+    # parameter only for pretty help.
+    async def verify(self, ctx: Context, member):
```
Comment (maxwai) [inline cogs/moderator.py:69]: Do not change unrelated lines.

```suggestion
    async def verify(self, ctx: Context, member):  # parameter only for pretty help.
```

**Review by maxwai — CHANGES_REQUESTED** [review]

**Comment by maxwai** [comment]
> It adds the option to use the `userID` instead of a mention for verifying. Its intent is to provide quality of life. I've already talked with @marcelropos about this change

Ok, then it is fine. Just try to include such information in the initial Pull Request Comment in the future.

Fix the remaining things and we are good.

I also changed the base Branch to develop as we also need to test it before deploying (it couldn't be done before as the develop branch was behind master sorry)

**Review by marcelropos — APPROVED** [review]
fix that on develop

**Comment by marcelropos** [comment]
fix that on develop
